arcade/scene.py

```diff
@@ -432,9 +432,9 @@ def update_animation(
     def draw(
         self,
         names: Optional[Iterable[str]] = None,
-        filter: Optional[int] = None,
+        filter: Optional[OpenGlFilters] = None,
         pixelated: bool = False,
-        blend_function: Optional[Union[Tuple[int, int], Tuple[int, int, int, int]]] = None,
+        blend_function: Optional[BlendFunctions] = None,
```

Comment:
It looks like `ArcadeContext` still has references to what look like tuple blend functions as class-level constants, so this may be a premature change.